flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -493,6 +497,59 @@ public boolean accept(File dir, String name) {
 		}
 	}
 
+	public static boolean verifyTokenKindInContainerCredentials(final String[] tokens, final String containerId) {
+		List<String> tokenList = Arrays.asList(tokens);
```

Comment:
if `tokens` is of type `Collection<String>`, then we would not need to convert it here.